SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -223,11 +223,20 @@ UAbstractLBStrategy* ULayeredLBStrategy::GetLBStrategyForVisualRendering() const
 {
 	// The default strategy is guaranteed to exist as long as the strategy is ready.
 	checkf(LayerNameToLBStrategy.Contains(SpatialConstants::DefaultLayer),
-		   TEXT("Load balancing strategy does not contain default layer which is needed to render worker debug visualization. "
+		TEXT("Load balancing strategy does not contain default layer which is needed to render worker debug visualization. "
 				"Default layer presence should be enforced by MultiWorkerSettings edit validation. Class: %s"),
 		   *GetNameSafe(this));
+	return GetLBStrategyForLayer(SpatialConstants::DefaultLayer);
+}
 
-	return LayerNameToLBStrategy[SpatialConstants::DefaultLayer];
+UAbstractLBStrategy* ULayeredLBStrategy::GetLBStrategyForLayer(FName Layer) const
+{
+	check(IsReady());
```

Comment:
I previously had to remove this check(IsReady()) because this function is now called in the Editor to visualise the worker boundaries. When this check was in before it fails when it is called from the Editor. To test this try to enable "Spatial Editor Debugger" it's a drop-down on the "Start Deployment" button.

Reply:
Could wrap it so it is not checked in the editor?
```
	#ifndef	WITH_EDITOR
	check(IsReady());
	#endif // WITH_EDITOR
```